cmeutils/tests/test_structure.py

```diff
@@ -49,6 +49,26 @@ def test_bond_not_found(self, p3ht_gsd):
         with pytest.raises(ValueError):
             bonds = bond_distribution(p3ht_gsd, "xx", "ss", start=0, stop=1)
 
+    def test_bond_histogram(self, p3ht_gsd):
+        bonds_hist = bond_distribution(
+                p3ht_gsd, "cc", "ss", start=0, stop=1, histogram=True
+        )
+        bonds_no_hist = bond_distribution(
+                p3ht_gsd, "cc", "ss", start=0, stop=1, histogram=False
+        )
+        assert bonds_hist.ndim == 2
+        assert bonds_no_hist.ndim == 1
```

Comment:
Might be worth checking that the bond distributions have reasonable values for these known systems.

Reply:
There are separate unit tests that test the values returned for these known systems. Do you think we need to check the values again here as well?

Reply:
OOPS. My bad--I mistakenly thought that the bond/angle distribution functions were _added_ in this PR.